src/components/BestSelling/BestSelling.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './BestSelling.css'
+import { Card } from '../index'
+import { useCart } from "../../context/CartContext"
+
+const BestSelling = (props) => {
+    const { cartState: { productsData } } = useCart();
```

Comment:
You can pass the value from context after destructuring so that you don't have to destructure everywhere you use `useCart`